tests/unit/resources/test_factory.py

```diff
@@ -188,13 +233,25 @@ def test_dangling_resource_create_with_kwarg(self):
             }
         }
 
-        resource = self.load('test', 'test', {}, defs, None)()
+        resource = self.load('test', 'test', model, defs, None)()
         q = resource.Queue(url='test')
 
         self.assertIsInstance(q, ServiceResource,
             'Dangling resource created with kwargs is not a ServiceResource')
 
     def test_dangling_resource_shares_client(self):
+        model = {
```

Comment:
If you are going to use the same model a bunch of times in the test. It may be easier to just add it in the the ``setUp()`` as a property of the test class and then override it in the test method if needed.